miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppStorage.kt

```diff
@@ -43,6 +47,25 @@ internal class MiniAppStorage(
     @VisibleForTesting
     fun getFileName(file: String) = urlToFileInfoParser.getFileName(file)
 
-    fun getSavePathForApp(appId: String, versionId: String) =
-        "${basePath.path}/$SUB_DIR_MINIAPP/$appId/$versionId"
+    @VisibleForTesting
+    internal fun getParentPathApp(appId: String) = "${basePath.path}/$SUB_DIR_MINIAPP/$appId/"
+
+    fun getSavePathForApp(appId: String, versionId: String) = "${getParentPathApp(appId)}$versionId"
```

Comment:
I think let's break the paths down and use them up, they will tend to look untidy soon.

```
basePathHostApp : basePath.path
rootPathForMiniApp : basePathHostApp/SUB_DIR_MINIAPP/
pathForMiniApp : rootPathForMiniApp/appId/
pathForMiniAppVersionDownload : pathForMiniApp/versionId